src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
+    },
+    minlength: {
+      "minlength": "Password minimum length is 8 characters ..."
+    }
   },
   Registration: {
-
+    mismatch: {
+      "match": "The passwords entered do not match."
+    },
+    error: {
+      "error": "Sorry, something went wrong. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    invalidUser: {
+      "account": "Whoops... Account not found. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    noPassword: {
+      "password": "Whoopes... Unable to register, invalid password."
+    },
+    alreadyRegistered: {
+      "registered": "This account has already registered. Please type your registered email and password to login."
+    }
   },
   ResetPassword: {
-
+    resetLoginFailed: {
+      "failed": "Whoops, login failed, please go to login page to sign in."
+    },
+    invalidLink: {
+      "invalid": "Oops... an error occurred, please try again. In 5 seconds you will return to the login page ..."
```

Comment:
This looks like an invalid password reset link. If so then 
"Oops... The link to log you in appears to be broken. In 5 seconds you will return to the login page where you can try again with your username and password."